src/utils/zcl_abapgit_timer.clas.testclasses.abap

```diff
@@ -19,9 +19,8 @@ ENDCLASS.
 CLASS ltcl_timer IMPLEMENTATION.
 
   METHOD setup.
-    DATA lv_skip TYPE c LENGTH 30.
-    GET PARAMETER ID 'TSE' FIELD lv_skip.
-    mv_disabled = boolc( sy-sysid = 'ABC' OR lv_skip CS 'SKIP_TIMER' ).
+* dont run on open-abap, the NodeJS garbage collector can run any time, causing flaky test
+    mv_disabled = boolc( sy-sysid = 'ABC' ).
```

Comment:
also exclude Docker image?  `or sy-sysid = 'A4H'` ?

Reply:
suggest just deleting those unit tests then? if we dont like to run them. I think the code it tests is not super complex, so tests can be deleted IMHO

Reply:
sure. let's delete the test class

Reply:
deleted